config/routes.rb

```diff
@@ -4,6 +4,8 @@
   namespace :api do
     post 'login', to: 'users#login'
     get 'feed', to: 'scores#user_feed'
-    resources :scores, only: %i[create destroy]
+    get 'scores', to: 'scores#user_feed'
+    # get '/scores', to: 'scores#index'
```

Comment:
If you think we're not going to use this code anytime soon, please remove it instead of commenting it.